pkgs/games/chromium-bsu/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchurl
+, SDL2
+, SDL2_image
+, SDL2_mixer
+, fontconfig
+, freealut
+, freeglut
+, ftgl
+, gettext
+, libGL
+, libGLU
+, openal
+, pkg-config
```

Comment:
This list should be sorted. It is ok if `lib` stays at the top, or if you divide this into groups (e.g. `lib,stdenv,fetchurl`, rest, or such) and the sort it alphabetically, but it should be sorted, especially, because it currently looks pretty random.

Reply:
But the list is `M-x sort-lines` sorted.

Reply:
ok.